lib/shopify_api/meta.rb

```diff
@@ -0,0 +1,16 @@
+
+# frozen_string_literal: true
+module ShopifyAPI
+  class Meta < ActiveResource::Base
```

Comment:
What is the default ActiveResource timeout? This endpoint is very fast normally, and we probably don't want to delay app startup by more than a second or two before just moving on.

Reply:
Or at least not delay startup without telling the developer what we are waiting on?

Reply:
We should still probably set an explicit timeout here